lisa/transformers/dom0_kernel_installer.py

```diff
@@ -124,10 +127,19 @@ def install(self) -> str:
                 node.get_pure_path(f"/boot/initrd.img-{new_kernel}"),
             )
         else:
+            # Mariner 2.0 initrd
+            target = f"/boot/initrd.img-{current_kernel}"
+            link = f"/boot/initrd.img-{new_kernel}"
+
+            if isinstance(node.os, CBLMariner) and mariner_version == 3:
+                # Mariner 3.0 initrd
+                target = f"/boot/initramfs-{current_kernel}.img"
+                link = f"/boot/initramfs-{new_kernel}.img"
```

Comment:
Can we make 3.0 's values as default? Most probably, next versions of mariner (4, 5 etc) would have same file paths as 3.0 and it would avoid us code changes.

```suggestion
            # Mariner 3.0 initrd
            target = f"/boot/initramfs-{current_kernel}.img"
            link = f"/boot/initramfs-{new_kernel}.img"

            if isinstance(node.os, CBLMariner) and mariner_version == 2:
                # Mariner 2.0 initrd
                target = f"/boot/initrd.img-{current_kernel}"
                link = f"/boot/initrd.img-{new_kernel}"
```

Reply:
You are right. If newer mariner version supposed to use the same grub file, it would make sense to use >=3 as default case. Let me check on this.

Reply:
Done please check